src/db/sysdb_ops.c

```diff
@@ -5006,3 +5006,85 @@ errno_t sysdb_mark_entry_as_expired_ldb_val(struct sss_domain_info *dom,
     talloc_free(tmp_ctx);
     return ret;
 }
+
+enum sysdb_entry_type {
+    TYPE_USER=0,
+    TYPE_GROUP
+};
```

Comment:
I agree with @fidencio I would prefer either adding a boolean or just reusing SYSDB_MEMBER_USER/SYSDB_MEMBER_GROUP or even `enum sysdb_obj_type` instead of adding yet another enum that is only used at one place.